x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/step_define_package_policy.test.tsx

```diff
@@ -99,45 +92,58 @@ describe('StepDefinePackagePolicy', () => {
         packagePolicy={packagePolicy}
         updatePackagePolicy={mockUpdatePackagePolicy}
         validationResults={validationResults}
-        submitAttempted={false}
+        submitAttempted={true}
       />
     ));
 
   beforeEach(() => {
     packagePolicy = {
       name: '',
       description: 'desc',
-      namespace: 'default',
+      namespace: undefined,
```

Comment:
nit: I think that usually the package policies have either `default` or a custom namespace, I don't think it's possible to have it undefined in reality. Did you want to test something related to this field?

Reply:
Thanks for clarifying it, I could see some integrations actually have empty package policy namespace, but not `undefined`, also they fallback to `default` when the namespace is empty, I updated the tests to reflect that.

**it should fallback to the default namespace when namespace is not set in package policy and there's no agent policy**

<img width="1667" alt="image" src="https://github.com/user-attachments/assets/036bc659-5a8c-47b3-81ce-d7b83c752adc">


**it should display namespace from agent policy when there's no package policy namespace**
<img width="1649" alt="image" src="https://github.com/user-attachments/assets/285bf11d-f17e-45f8-964a-e369eb9fcc21">


**it should display vars coming from package policy**
<img width="1250" alt="image" src="https://github.com/user-attachments/assets/468e7019-83e6-4169-ac43-9651f27a2181">